examples/CMakeLists.txt

```diff
@@ -17,8 +17,9 @@ set(ENV{PKG_CONFIG_PATH} "${PKG_CONFIG_CHANGED_PATH}")
 
 find_package(LibtorrentRasterbar REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
+find_package(OpenSSL REQUIRED)
```

Comment:
Encryption is an option in the top-level CMakeLists file. Examples are built independently, so they don't inherit that. The examples use SSL if libtorrent is built with that support. There's not a good way to detect whether libtorrent was built with OpenSSL from the CMake level, so the simplest may be to make it an optional component.

There's some weird stuff going on in https://github.com/arvidn/libtorrent/blob/master/examples/cmake/FindLibtorrentRasterbar.cmake as well. I can dig further for what's the right thing to do.